packages/squiggle-lang/__tests__/TS/Scalars_test.ts

```diff
@@ -0,0 +1,51 @@
+import { errorValueToString } from "../../src/js/index";
+import { testRun } from "./TestHelpers";
+import * as fc from "fast-check";
+
+describe("Scalar manipulation is well-modeled by javascript math", () => {
+  test("in the case of logarithms (with assignment)", () => {
+    fc.assert(
+      fc.property(fc.float(), (x) => {
+        let squiggleString = `x = log(${x}); x`;
```

Comment:
why use assignment, instead of just ``log(${x})``

Reply:
I forget what my reasoning was but you're probably right that I'd like this test not to fail if something in the interpreter/parser messes up assignment some day.

Reply:
My concern is more that it unnecessarily complicates it, for readers of the test